modules/core/plan/modules/record/src/Entity/PlanRecord.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Drupal\plan_record\Entity;
+
+use Drupal\Core\Entity\ContentEntityBase;
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+
+/**
+ * Defines the Plan record relationship entity.
+ *
+ * @ContentEntityType(
+ *   id = "plan_record",
+ *   label = @Translation("Plan record relationship"),
+ *   bundle_label = @Translation("Plan record relationship type"),
+ *   label_collection = @Translation("Plan record relationships"),
+ *   label_singular = @Translation("plan record relationship"),
+ *   label_plural = @Translation("plan record relationships"),
+ *   label_count = @PluralTranslation(
+ *     singular = "@count plan record relationship",
+ *     plural = "@count plan record relationships",
+ *   ),
+ *   base_table = "plan_record",
```

Comment:
should this have a `data_table = "plan_record_data"` ?

Reply:
Yes! Done: https://github.com/farmOS/farmOS/pull/781/commits/9d3dd2746909feb90bbd1a882dd7986fd62591c9